tests/test_point_cloud_format.py

```diff
@@ -0,0 +1,412 @@
+import os.path as osp
```

Comment:
This file has multiple typos in `occluded` (occ **u** luded)

Reply:
Resolved